com.sap.cloud.lm.sl.cf.core/src/test/resources/com/sap/cloud/lm/sl/cf/core/cf/v1/mtad-12.yaml

```diff
@@ -0,0 +1,9 @@
+_schema-version: "1.0.0"
+ID: com.sap.sample.mta
+version: 0.1.0
+
+modules:
+  - name: foo
+    type: java.tomee
+    properties:
+      route: foo-custom-route.test-domain
```

Comment:
Add tests for "routes" and also with different contnent, such as tcp-example.com:1234, www.example.com/foo

Reply:
done